Modules/CustomerHistory/Frames.lua

```diff
@@ -354,16 +354,7 @@ function CraftSim.CUSTOMER_HISTORY.FRAMES:UpdateCustomerHistoryList()
     customerList:UpdateDisplay(
         function(rowA, rowB)
             if rowA.customerHistory.totalTip and rowB.customerHistory.totalTip then
-                if rowA.customerHistory.totalTip > rowB.customerHistory.totalTip then
-                    return true
-                end
-                if rowA.customerHistory.totalTip < rowB.customerHistory.totalTip then
-                    return false
-                end
-                if rowA.customerHistory.totalTip == rowB.customerHistory.totalTip then
-                    return true
-                end
-                return false
+                return rowA.customerHistory.totalTip < rowB.customerHistory.totalTip
```

Comment:
you want to flip this to preserve old ordering (highest tip on top)
and this is still broken for equally tipping customers, you need some fallback (comparing names for example)